frontend/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "portal",
+  "version": "0.1.0",
+  "private": true,
+  "description": "",
+  "keywords": [],
+  "author": "The Opencast project",
```

Comment:
Should this have our names, like the `Cargo.toml`?

Reply:
:shrug: